app/assets/javascripts/covid19/src/webapp/components/structures-table/DetailsDialog.tsx

```diff
@@ -34,36 +34,33 @@ export const DetailsDialog: React.FC<DetailsDialogProps> = React.memo(props => {
                 </ImgContainer>
 
                 <div>
-                    <ModifiedAccordion expandedAccordion={expandedAccordion} field="entities">
-                        {row.entities.length > 0 ? (
-                            <EntityCell data={data} row={row} moreDetails={false}></EntityCell>
-                        ) : (
-                            <NoCellsFound name="entities" />
-                        )}
+                    <ModifiedAccordion
+                        expanded={expandedAccordion}
+                        field="entities"
+                        isEmpty={_.isEmpty(row.entities)}
+                    >
+                        <EntityCell data={data} row={row} moreDetails={false}></EntityCell>
                     </ModifiedAccordion>
```

Comment:
[no need to change] You could also pass the row and the component will check if `row[field]` is Empty.

Reply:
Actually I like it because I don't see `isEmpty` prop really necessary, I would like to change it.